app/phone-list/phone-list.template.html

```diff
@@ -3,9 +3,11 @@
     <h1>Playground</h1>
     <div ui-grid=gridOptions class="mb-3"></div>
     <button class="btn btn-default" ng-click="clear()">Clear All Filters</button>
-    <button class="btn btn-default" ng-click="toggleColumns()">Show/Hide Columns</button>
-    <div ng-repeat="menuItem in menuItems">
-        <input type="checkbox" value="{{menuItem.title}}" name="menuItem{{$index + 1}}" id="menuItem{{$index + 1}}">
-        <label for="menuItem{{$index + 1}}">{{menuItem.title}}</label>
+    <button class="btn btn-default" ng-click="toggleMenu()">Show/Hide Columns</button>
+    <div ng-show="canViewMenu">
+        <div ng-repeat="menuItem in menuItems">
+            <input type="checkbox" value="{{menuItem.title}}" name="menuItem{{$index + 1}}" id="menuItem{{$index + 1}}">
```

Comment:
For now, we'll use `menuItem{{$index + 1}}`, but this may change as we update this to be functional.